staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -181,6 +200,23 @@ func (f *ConfigFlags) ToRawKubeConfigLoader() clientcmd.ClientConfig {
 	return clientConfig
 }
 
+// toPersistentRawKubeConfigLoader binds config flag values to config overrides
+// Returns a persistent clientConfig for propagation.
+func (f *ConfigFlags) toPersistentRawKubeConfigLoader() clientcmd.ClientConfig {
+	if f.clientConfig != nil {
+		return f.clientConfig
```

Comment:
Do not use double check locks in go.  They don’t work and are unnecessary.

Reply:
@smarterclayton Done